flows/triggers/testdata/campaign.json

```diff
@@ -57,9 +57,11 @@
         },
         "events": [],
         "context": {
-            "keyword": null,
-            "params": null,
-            "type": "campaign"
+            "keyword": "",
```

Comment:
I've convinced myself that distinguishing between `null` and `""` isn't helpful.. and is a safe change to make since they both render to empty strings. The type of the trigger is in the context so if you need to split on that you can.. you don't need to compare `null` vs `""`